test/pages/CheckYourAnswersPageSpec.scala

```diff
@@ -18,17 +18,33 @@ package pages
 
 import controllers.routes
 import models.BusinessType.*
+import models.RegistrationType.ThirdParty
 import models.registration.responses as registrationResponses
 import models.subscription.requests.SubscriptionRequest
-import models.subscription.{IndividualContact, responses as subscriptionResponses}
-import models.{NormalMode, RegistrationType, SubscriptionDetails, UserAnswers}
+import models.subscription.responses.AlreadySubscribedResponse
+import models.subscription.{IndividualContact, Organisation, OrganisationContact, responses as subscriptionResponses}
+import models.{BusinessType, NormalMode, RegistrationType, SubscriptionDetails, UserAnswers}
 import org.scalacheck.Gen
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.must.Matchers
 import org.scalatest.{OptionValues, TryValues}
 
 class CheckYourAnswersPageSpec extends AnyFreeSpec with Matchers with TryValues with OptionValues {
 
+  private def getDetails(businessType: BusinessType) =
```

Comment:
It is better to create a builder so that you have nice default values. SubscriptionDetails object is created a few times in your tests and if you have a builder, you can save quite a bit of code writing.

Reply:
Left as is as the method I added is meant only t for tests and I don't want to put test-only stuff in production code.

Reply:
Test Builders live in the tests, they are not production code.